crates/matrix-sdk-crypto/src/session_manager/group_sessions/snapshots/prepare_machine_with_dehydrated_device_of_verification_violation_user.snap

```diff
@@ -1,6 +1,7 @@
 ---
 source: crates/matrix-sdk-crypto/src/session_manager/group_sessions/share_strategy.rs
 expression: ruma_response_to_json(keys_query.clone())
+snapshot_kind: text
```

Comment:
do you know where these extra `snapshot_kind` lines have come from? they seem incorrect.

Reply:
I just ran only both of `cargo insta test` and `cargo insta review` in according to your guide.
Can you please try on your side and let me know that result?

Reply:
Yes, I did already.

I replicated your changes to `crates/matrix-sdk-crypto/src/session_manager/group_sessions/share_strategy.rs` and ran `cargo insta test`. It did not add `snapshot_kind` (or, indeed, make any changes to the snapshot files other than to their names).

Reply:
Should I remove `snapshot_kind` manually?

Reply:
Fixed by https://github.com/matrix-org/matrix-rust-sdk/pull/4625/commits/b2273ec72a03f8edc26a2e9b098f2b9e764473ea